examples/evaluators/golang/simple/Dockerfile

```diff
@@ -1,11 +1,10 @@
-# Golang application builder steps
-FROM gcr.io/open-match-public-images/openmatch-base:dev as builder
-WORKDIR /go/src/github.com/GoogleCloudPlatform/open-match/examples/evaluators/golang/simple
-COPY . .
-RUN go get -d -v
+FROM open-match-base-build as builder
+
+WORKDIR /go/src/github.com/GoogleCloudPlatform/open-match/examples/evaluators/golang/simple/
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo .
 
-FROM gcr.io/distroless/static
-COPY --from=builder /go/src/github.com/GoogleCloudPlatform/open-match/examples/evaluators/golang/simple .
+FROM gcr.io/distroless/static  
+COPY --from=builder /go/src/github.com/GoogleCloudPlatform/open-match/examples/evaluators/golang/simple/simple .
+COPY --from=builder /go/src/github.com/GoogleCloudPlatform/open-match/config/matchmaker_config.yaml ./config/
```

Comment:
Is this really needed?

Reply:
Ah - I remember seeing this go away in a later review. (just a symlink right)

Reply:
Added a TODO for this.